client/app/views/partials/tip_submission_state.html

```diff
@@ -26,9 +26,10 @@
                             <label>
                                 <span data-translate>Substate: </span>
                             </label>
-                            <select id="assignSubmissionSubState" ng-model="tip.selectedSubmissionSubState" ng-options="substate as substate.label for substate in ::tip.selectedSubmissionState.substates">
+                            <select id="assignSubmissionSubState" ng-model="tip.selectedSubmissionSubState" ng-options="substate as substate.label for substate in tip.selectedSubmissionState.substates">
                                 <option value=""><span data-translate>(None)</span></option>
                             </select>
+                            </select>
```

Comment:
@NCommander: this seems a repetition from what i see here

Reply:
Which IDE are you using? With globaleaks i'm using pycharm that works great with both python, javascript and angularjs specifically